.pre-commit-config.yaml

```diff
@@ -27,6 +27,13 @@ repos:
           # Configure Black to support only syntax supported by the minimum supported Python version in setup.py.
           - --target-version=py37
         files: \.py$|\.pyi$
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        args:
+          - --filter-files
+        exclude: ^lib/streamlit/__init__\.py$
```

Comment:
I think we can also skip files in the isort config, see [here](https://pycqa.github.io/isort/docs/configuration/options.html#skip).

Reply:
I added `# isort: skip_file` comment to this file, so it should be skipped by IDE tools.